flocker/apiclient/test/test_client.py

```diff
@@ -496,6 +496,29 @@ def got_result(container):
             d.addCallback(got_result)
             return d
 
+        def test_container_state(self):
+            """
+            ``list_containers_state`` returns information about state.
+            """
+            expected, d = create_container_for_test(self, self.client)
+
+            d.addCallback(lambda _ignored: self.synchronize_state())
+
+            d.addCallback(lambda _ignored: self.client.list_containers_state())
+
+            d.addCallback(
+                lambda containers: [(c.node_uuid, c.name) for c in containers]
+            )
+
+            d.addCallback(
+                lambda containers: self.assertIn(
+                    (expected.node_uuid, expected.name),
+                    containers
```

Comment:
I made some simple changes to make these attributes testable. So, the test now checks everything.